tidb-cloud/connect-to-tidb-cluster-serverless.md

```diff
@@ -5,29 +5,56 @@ summary: Learn how to connect to your TiDB Serverless cluster via different meth
 
 # Connect to Your TiDB Serverless Cluster
 
-This document introduces the methods to connect to your TiDB Serverless cluster.
+This document describes how to connect to your TiDB Serverless cluster.
 
 > **Tip:**
 >
 > To learn how to connect to a TiDB Dedicated cluster, see [Connect to Your TiDB Dedicated Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
 
+## Connection methods
+
 After your TiDB Serverless cluster is created on TiDB Cloud, you can connect to it via one of the following methods:
 
-- [Connect via private endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) (recommended)
+- Direct connections
+
+  TiDB Serverless is MySQL-compatible. You can connect to your TiDB Serverless cluster using any tool that supports MySQL connections, such as [MySQL client](https://dev.mysql.com/downloads/shell/).
+
+- [Data Service (beta)](/tidb-cloud/data-service-overview.md)
+
+  TiDB Serverless provides a Data Service feature that enables you to access TiDB Serverless data via an HTTPS request using a custom API endpoint.
+
+- [Serverless Driver (beta)](/tidb-cloud/serverless-driver.md)
+
+  TiDB Serverless provides Serverless Driver, a javascript driver that allows you to connect to your TiDB Serverless cluster in edge environments over HTTP.
+
+- [Chat2Query (beta)](/tidb-cloud/explore-data-with-chat2query.md)
+
+  TiDB Cloud is powered by artificial intelligence (AI). You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
+
+  In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
+
```

Comment:
I think at the end of this part, we need a short conclusion comparing these methods: direct connections/serverless driver/data service.